doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -113,11 +111,11 @@ See `Zigbee samples`_ for the list of changes for the Zigbee samples.
 ESB
 ---
 
-  * Added support for front-end modules.
-  * The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
-  * The number of PPI/DPPI channels used is increased from 3 to 6.
-  * Assigned events 6-7 from the EGU0 instance to the ESB module.
-  * Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
+* Added support for front-end modules.
+* The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
+* The number of PPI/DPPI channels used is increased from 3 to 6.
+* Assigned events 6-7 from the EGU0 instance to the ESB module.
+* Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
```

Comment:
```suggestion
* Added support for front-end modules.
  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.

* Updated:

  * Number of PPI/DPPI channels used from 3 to 6.
  * Events 6-7 from the EGU0 instance by assigning them to the ESB module.
  * The type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
```
This line sounds like part of the support for FEM? Anyway, it has a formatting that is off for release notes.
```
* The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
```